trainingbreakdown/databases/README.md

```diff
@@ -51,4 +51,4 @@
 
 ## Unit Lesson Plans
 
-For details of what is cover each session of in each lesson plan take a look [here](lessonplan)
+For details of what is cover each session of in each lesson plan take a look [here](lesson-plan)
```

Comment:
I haven't added the database lesson-plan folder yet, so the link goes to a 404. But it will work once I've added the folder